app/src/main/java/org/mozilla/tv/firefox/utils/Settings.kt

```diff
@@ -91,18 +91,7 @@ class Settings private constructor(context: Context) {
     val trackingProtectionPolicy: TrackingProtectionPolicy
         get() {
             return if (isBlockingEnabled) {
-                TrackingProtectionPolicy.select(
-                    // We want to use TrackingCategory.RECOMMENDED but it doesn't block ads properly:
-                    // a-c#4191. We write out the values in RECOMMENDED manually below and it works
-                    // properly.
-                    trackingCategories = arrayOf(
-                        TrackingCategory.AD,
-                        TrackingCategory.ANALYTICS,
-                        TrackingCategory.SOCIAL,
-                        TrackingCategory.TEST
-                    )
-                    // We use the default cookiePolicy.
-                )
+                TrackingProtectionPolicy.recommended()
```

Comment:
This adds the following tracking protection categories:
- Mozilla_social
- Cryptomining
- Fingerprinting

Adding tracking protection categories adds some risk that sites users are trying to use will break: can you follow up with Ashley that this is an acceptable risk for the upcoming release? We could postpone it to the following release though we might want to wait even another release if we're intending to take on the updated Disconnect lists from a-c next release, to make it clearer what might have caused the errors.

Reply:
@athomasmoz

Reply:
Let's wait until next release!